WhatBackend.EmailSendingService/WhatBackend.EmailSendingService.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Place this .sln into WhatBackend\WhatBackend.EmailSendingService.sln. It's more convenient to have solution in projects root directory.